.github/workflows/nodejs.yml

```diff
@@ -1,10 +1,18 @@
-name: Node.js CI
+name: Node.js CI / Scalingo CD
 
 on:
   push:
     branches: ["master"]
   pull_request:
     branches: ["master"]
+  schedule:
+    # https://crontab.guru/#0_2_*_*_*
+    - cron: "0 2 * * *"
```

Comment:
> Scheduled workflows run on the latest commit on the default or base branch.

[Source](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule) 

C'est bien ce qu'il nous faut 👍